mainhandler/imageregistryhandler.go

```diff
@@ -331,7 +341,9 @@ func (registryScan *registryScan) setImageToTagsMap(repo string, reporter datast
 				}
 			}
 		}
-		registryScan.mapImageToTags[registryScan.registry.hostname+"/"+repo] = tags
+		if len(tags) > 0 {
```

Comment:
Why don't put empty list of tags?
How the user will know they have this repo in the registry if we are ommiting it?